packages/create-sitecore-jss/src/bin.ts

```diff
@@ -6,13 +6,23 @@ import { initRunner } from './init-runner';
 import minimist, { ParsedArgs } from 'minimist';
 import { getAllTemplates, getBaseTemplates } from './common';
 
+const proxyAppMatcher = /node-.+-proxy/g;
+
 export const parseArgs = (): ParsedArgs => {
   // parse any command line arguments passed into `init sitecore-jss`
   // to pass to the generator prompts and skip them.
   // useful for CI and testing purposes
   const options = {
     boolean: ['appPrefix', 'force', 'noInstall', 'yes', 'silent', 'prePushHook'],
-    string: ['appName', 'destination', 'templates', 'hostName', 'fetchWith', 'language'],
+    string: [
+      'appName',
+      'destination',
+      'nodeAppDestination',
```

Comment:
I think we should change this to `proxyDestination` for the CLI param for clarity and consistency with prompts (and technically even the main app is a node app).

Reply:
`proxyDestination` reads like a request destination. Let me use `proxyAppDestination` then